src/SchemaUtils.ts

```diff
@@ -249,12 +264,46 @@ function parseStepWithSelectors(type: string, step: object): StepWithSelectors {
   };
 }
 
+function parseClickAttributes(step: object): ClickAttributes {
+  const attributes: ClickAttributes = {
+    offsetX: parseNumber(step, 'offsetX'),
+    offsetY: parseNumber(step, 'offsetY'),
+  };
+  const deviceType = parseOptionalString(step, 'deviceType');
+  if (deviceType && !pointerDeviceTypes.has(deviceType)) {
+    throw new Error(
+      `'deviceType' for click steps must be one of the following: ${[
+        ...pointerDeviceTypes,
+      ].join(', ')}`
+    );
+  }
+  attributes.deviceType = deviceType as ClickAttributes['deviceType'];
```

Comment:
could you please add a unit test for the schema utils parsing?

Reply:
Done.